### DISCUSSION TITLE
Is it possible to call Typst format just PDF?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#8596] [Q&A] [answered]
### Description

When you generate a manuscript that besides html also generate a typst built pdf, in "Other Formats" it shows "Typst".
![image](https://github.com/quarto-dev/quarto-cli/assets/392879/b1667ff5-6d64-44a9-9bc5-7e1ab9f30a84)

Would it be possible to just call it "PDF"?

**Comment by cscheid** [comment, marked as answer]
It's unfortunate, but "PDF" is what we use for the PDFs generated with LaTeX. I don't think that's currently configurable.

**Reply by fredguth** [reply]
You can not only have the same content in different file formats, but also different "template formats" (the same article in 2 different journal formats, for example).  Maybe this is a feature idea for the future.

**Reply by cscheid** [reply]
For different formats, I believe that we do use the more specific name. The unfortunate part is that we use the generic "PDF" for the case of LaTeX generated PDFs from our base format.

**Reply by mcanouil** [reply]
> You can not only have the same content in different file formats, but also different "template formats" (the same article in 2 different journal formats, for example). Maybe this is a feature idea for the future.

I don't really understand this sentence (wording is a bit odd), but you can do the following:

```yaml
format:
  pdf+myjournal: default
  pdf+yourjournal: default
  typst: default
  typst+journal: default
```

Additionally, you can set `output-file`.

**Reply by cscheid** [reply]
The comment is about the "Other formats" column, how we render the names, and how ideally those names should be different and configurable.
